package.json

```diff
@@ -53,12 +54,20 @@
     "start-hoc": "cd packages/light-hoc && yarn start",
     "start-react": "cd packages/fether-react && yarn start",
     "start-ui": "cd packages/fether-ui && yarn start",
-    "test": "lerna run test --parallel"
+    "test": "CI=true lerna run test --parallel"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "FILES=`git diff --staged --name-only HEAD | grep -E '.js$|.ts$'`; [ -z \"$FILES\" ] && exit 0; yarn lint-files $FILES; git add $FILES",
```

Comment:
I'm not sure what is better: showing the user an error "Please fix your linting errors", or silently fix them for the user. We can keep it like this.

Btw would `yarn lint-files '**/*.{js,ts}'; git add '**/*.{js,ts}'` here?